src/client.c

```diff
@@ -49,10 +49,12 @@ PgDatabase *prepare_auth_database(PgSocket *client)
 	PgDatabase *auth_db = NULL;
 	const char *auth_dbname = client->db->auth_dbname ? client->db->auth_dbname : cf_auth_dbname;
 
-	if (!auth_dbname)
-		return client->db;
+    if (!auth_dbname) {
```

Comment:
Please double check the indentations, I think there is a tab/spacing issue.

Reply:
As I understand we must use tab indentation, if so I have to check the all changes in the PR. I'll do this in the next commit

Reply:
Done